### DISCUSSION TITLE
进入另一个玩家的房间，该如何获取这个玩家的相关数据

### DISCUSSION THREAD

**Discussion by zywj** [kbengine/kbengine#1571] [Q&A] [answered]
目前以副本的形式运行各自玩家的房间，但是对方的玩家的房间里的一些数据不知道该如何获取

**Comment by shangdibaozi** [comment, marked as answer]
写个接口发到客户端？
或者利用ALL_CLIENTS标签去设置相关的属性。

**Reply by zywj** [reply]
感谢回复。我现在是进入其他玩家房间，要生成这个玩家所属的一些的道具/装饰之类的，kbe 是以实体的形式交互的，我不知道如何正确创建实体
